pkg/generate/app/componentresolvers.go

```diff
@@ -90,58 +89,22 @@ func (r PerfectMatchWeightedResolver) Resolve(value string) (*ComponentMatch, er
 					}
 				}
 			}
-		}
-		return nil, ErrNoMatch{value: value}
+		}*/
```

Comment:
is this getting uncommented prior to merge?  or hoping for @rhcarvalho's changes?